web_viewer/client/control.ts

```diff
@@ -53,13 +53,25 @@ export class FirstPersonController {
     this.phiDelta = 0;
 
     window.addEventListener(
-        'keydown', event => this.onKeyDown(<KeyboardEvent>event), false);
+      'keydown',
+      (event) => this.onKeyDown(<KeyboardEvent>event),
+      false
```

Comment:
Nit: not for this PR, but I think you can just leave the `false` out here (and at least in JS, that argument is typically omitted unless it's `true`).